ntbs-service/Models/ClinicalDetails.cs

```diff
@@ -9,11 +9,15 @@ namespace ntbs_service.Models
     [Owned]

```

Comment:
Shouldn't we set SymptomOnsetDate to null somewhere in here if the boolean flag is not set?

Reply:
Realised I left it out but on ClinicalDetails.cshtml.cs I'll set it to null like with the rest of the dates associated with a flag